releasenotes/notes/add-parameters-to-template-substitution-a1379cdbfcc10b5c.yaml

```diff
@@ -0,0 +1,88 @@
+---
+features:
+  - |
+    The Parameter class is now allowed in template optimization if the
+    parameterized gates are RZXGates and RXGates. It has been removed from 
```

Comment:
By `Parameter` we probably mean `ParameterExpression`, I think - that's the more general case.  These can also be Sphinx cross-referenced: `` :class:`.ParameterExpression` ``, ``:class:`.RZXGate`\ s``, etc.

Reply:
Added the cross-referencing